test/chromedriver-specs.js

```diff
@@ -28,12 +27,15 @@ function nextError (cd) {
 }
 
 async function assertNoRunningChromedrivers () {
-  // let res = await Q.nfcall(psNode.lookup, {command: /([^ ]*)chromedriver$/,
-  //                                          psargs: 'aux'});
-  let f = B.promisify(psNode.lookup);
-  let res = await f({command: /([^ ]*)chromedriver$/, psargs: 'aux'});
+  let {stdout} = await exec('ps', ['aux']);
```

Comment:
Does `ps-node` translate for Windows? If so, then this will not be a drop-in replacement.